solr/config/solrconfig.xml

```diff
@@ -407,4 +407,9 @@
     </arr>
   </requestHandler>
 
+  <updateHandler class="solr.DirectUpdateHandler2">
+    <updateLog>
+      <str name="dir">${solr.core0.data.dir:}</str>
+    </updateLog>
+  </updateHandler>
```

Comment:
This change is needed for the atomic updates to work. This setting is [already enabled in our production/staging configuration](https://github.com/pulibrary/pul_solr/blob/main/solr_configs/cicognara/conf/solrconfig.xml#L410-L414), but it was missing in the local development copy.